digdag-core/src/test/java/io/digdag/core/workflow/IfOperatorFactory.java

```diff
@@ -0,0 +1,68 @@
+package io.digdag.core.workflow;
```

Comment:
Why did you add this class instead of only modifying `io.digdag.standards.operator.IfOperatorFactory`?

Reply:
I am sorry for my late response.
The reason why I added the test class is that some other tests also add its test classes  in digdag-core/src/test/java/io/digdag/core/workflow such as EchoOperatorFactory, FailOperatorFactory and so on.
I adjusted to such manner.
I guess if I use the class in digdag-standards to test digdag-core, digdag-standarads is forced to depend on digdag-standards. And it may not be preferred.
But I don't know detail, so if I should use digdag-standards, I change the code.
Best regards,
  You Yamagata